public/app/features/chat/Chat.tsx

```diff
@@ -0,0 +1,293 @@
+import React, { FunctionComponent, PureComponent, useState } from 'react';
+import { TextArea, IconButton, WithContextMenu, MenuItem, MenuGroup } from '@grafana/ui';
+import { getGrafanaLiveSrv, getBackendSrv } from '@grafana/runtime';
+import { isLiveChannelMessageEvent, LiveChannelScope } from '@grafana/data';
+import { Unsubscribable } from 'rxjs';
+import { marked } from 'marked';
+import { sanitize } from '@grafana/data/src/text/sanitize';
+
+export interface ChatProps {
+  contentTypeId: number;
+  objectId: string;
+  actions?: ChatMessageAction[];
+}
+
+export interface User {
+  id: number;
+  name: string;
+  login: string;
+  email: string;
+  avatarUrl: string;
+}
+
+export interface Message {
+  id: number;
+  content: string;
+  created: number;
+  userId: number;
+  user: User;
+}
+
+export interface MessagePacket {
+  event: string;
+  messageCreated: Message;
+}
+
+export interface ChatState {
+  messages: Message[];
+  value: string;
+}
+
+function renderChatMarkdown(str?: string): string {
+  const html = marked(str || '', {
```

Comment:
rather than use marked directly, should should likey just use:
```
import { renderTextPanelMarkdown } from '@grafana/data';
```
and respect the sanatize flag from `config.disableSanitizeHtml`  see:

https://github.com/grafana/grafana/blob/main/public/app/plugins/panel/text/TextPanel.tsx#L47

Reply:
Existing funcs use the following markdown options:

```
const markdownOptions = {
  pedantic: false,
  gfm: true,
  smartLists: true,
  smartypants: false,
  xhtml: false,
};
```

In live discussions we use the following currently:

```
{
    pedantic: false,
    gfm: true,
    smartLists: true,
    smartypants: false,
    xhtml: false,
    breaks: true,
}
```

I.e. adding `breaks: true` to correctly render line breaks.